### ISSUE TITLE
Various File renamer problems

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dizzydezz/jmm#369]
```
How would one write the script syntax,"If Group ID is NOT unknown AND Anime ID 
is NOT unknown, DO Replace filename with original filename"?
Can't seem to figure it out.

-----------
It should be something like

IF I(aid);I(gid) DO ADD '%sna'
OR
IF I(sna) DO ADD '%sna'

But neither work
--------------

IF G(!unknown);A(!unknown) DO REPLACE '*' '%sna'
but nothing happened when I previewed the files.

```

Original issue reported on code.google.com by `werndly...@gmail.com` on 1 Jan 2013 at 1:40

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 3.1.28
```

Original comment by `werndly...@gmail.com` on 7 Jan 2013 at 5:40
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Sorry, I know this isn't really the right place to ask this.
Where does JMM get the %sna value? If it's from AniDB, how does one add an 
original file name where one is missing; because, for example:
Hunter x Hunter 2011 episode 4 by Tsuki in 720p ( 
http://anidb.net/perl-bin/animedb.pl?show=addfile&aid=&eid=133906&edit=983787 ) 
does not rename properly, but
Hunter x Hunter 2011 episode 5 by Tsuki in 720p ( 
http://anidb.net/perl-bin/animedb.pl?show=addfile&aid=&eid=133905&edit=986649 ) 
renames to the proper original file name, but I can't find where to add the 
proper file name for episode 4 on AniDB, so that JMM can get it & rename 
properly.
```

Original comment by `inferk...@gmail.com` on 9 Jul 2013 at 11:17
